share/ci/compiler_hipcc.yml

```diff
@@ -10,6 +10,8 @@
     #  "Could NOT find MPI (missing: MPI_C_FOUND MPI_CXX_FOUND)"
     PIC_CMAKE_ARGS: "-DMPI_CXX_WORKS=ON -DMPI_CXX_VERSION=0"
     # ISAAC is not working with HIP
```

Comment:
not part of the PR: Is this still true?

Reply:
Should work and should be checked separately.

Reply:
Okay - I opened #4714